src/ecalc/libraries/libecalc/common/tests/integration/snapshots/test_consumer_system_v2/test_compressor_system_v2_results/consumer_system_v2.json

```diff
@@ -18,6 +18,76 @@
                         0.0
                     ]
                 },
+                "id": "09c1b79cf6cf1ef0773a194679246a4a",
```

Comment:
Imo. it is "impossible" to compare with the old setup and this wrt the old setup that had fuel consumers, and should in little degree be affected.

Is it to much to ask for to keep the old and also introduce the el consumer system additionally, in order to see how the fuel consumer system was affected? 

Also, I assume this is a step towards comparing the backwards correctness with v1, and not fully, because I cannot see the v1 results or how that is compared...but I may miss out of something :)

Reply:
The specific comparison is made in the integration test not in the snapshot.